modules/util/path.go

```diff
@@ -14,3 +17,15 @@ func EnsureAbsolutePath(path string, absoluteBase string) string {
 	}
 	return filepath.Join(absoluteBase, path)
 }
+
+// GetDirectorySize returns the dumb disk consumption for a given path
+func GetDirectorySize(path string) (int64, error) {
+	var size int64
+	err := filepath.Walk(path, func(_ string, info os.FileInfo, err error) error {
+		if info != nil && !info.IsDir() {
```

Comment:
Just in case, I'd test for regular files. In Linux you can get stuck here if you encounter a pipe or some character or block device (e.g. `mknod p mystuckypipe`).
The _real_ complication comes if we want to check for circular references. But I wouldn't bother.

Reply:
To ensure it's a regular file, [the check goes like this](https://golang.org/pkg/os/#FileMode):
```
const notRegular := os.ModeDir | ModeSymLink | os.ModeNamedPipe | os.ModeSocket | os.ModeDevice | os.ModeCharDevice | ModeIrregular
if info != nil && (info.FileMode & notRegular) == 0 {
```

Reply:
done